website/mainSearchPage/index.html

```diff
@@ -242,12 +242,22 @@ <h2 class="featurette-heading">
       </footer>
     </div>
 
+
+
+    <!------------------search bar calendar---------------->
+
+
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
```

Comment:
well, its part of bootstrap framework, and we are using it. could ended up having side affects if not there.